debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -121,6 +121,9 @@ public SqlServerConnection(Configuration config, Clock clock, SourceTimestampMod
         boolean supportsAtTimeZone = supportsAtTimeZone();
         transactionTimezone = retrieveTransactionTimezone(supportsAtTimeZone);
         lsnToTimestamp = getLsnToTimestamp(supportsAtTimeZone);
+        getAllChangesForTable = supportsAtTimeZone
+            ? "SELECT sys.fn_cdc_map_lsn_to_time([__$start_lsn]) AT TIME ZONE 'UTC', * FROM cdc.[fn_cdc_get_all_changes_#](?, ?, N'all update old')"
```

Comment:
Would it be possible to put the timestamp to the last column instead of the first one? This way, the existing column positions won't have to change. Especially important if selecting the timestamp conditionally based on the configuration.

Reply:
The additional column will need to be selected only if required by the configuration.